examples/with-react-apollo/src/lib/apollo.js

```diff
@@ -0,0 +1,26 @@
+import { ApolloClient } from "apollo-client";
+import { createHttpLink } from "apollo-link-http";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import fetch from "isomorphic-unfetch";
+
+export function createClient() {
+  const client = new ApolloClient({
+    connectToDevTools: process.browser,
+    ssrMode: !process.browser,
+    ssrForceFetchDelay: 100,
+    link: createHttpLink({
+      uri: "https://metaphysics-production.artsy.net/",
+      fetch
+    }),
+    defaultOptions: {
+      query: {
+        fetchPolicy: "cache-and-network"
```

Comment:
query's fetchPolicy does not have "cache-and-network". watchQuery on the other hand does contain "cache-and-network".